.nvmrc

```diff
@@ -0,0 +1 @@
+v12.18.3
```

Comment:
It's nice having this define a consistent version of Node for development. I can remove if you don't want it

Reply:
This only has an effect for people that use `nvm` and might hide issues in future versions of node so I'd rather you revert this

Reply:
Sounds good. I didn't know what version of Node to use, so thought it might be helpful. Which version are you developing on?

Reply:
v12.18.3